samples/attested_tls/client/enc/cert_verifier.cpp

```diff
@@ -48,18 +48,20 @@ int cert_verify_callback(
     if (cert_size <= 0)
         goto exit;
 
-    result = oe_verify_attestation_certificate(
-        cert_buf, cert_size, enclave_identity_verifier_callback, NULL);
+    oe_verifier_initialize();
```

Comment:
Move to somewhere else so it only gets called once.

Reply:
Updated